README.md

```diff
@@ -48,7 +48,7 @@ Here's to a thrilling Hacktoberfest voyage with us! 🎉
 # Decentralized Web Node (DWN) SDK <!-- omit in toc -->
 
 Code Coverage
```

Comment:
On code coverage: I noticed that there are a few placed related to roles (and grants) that missed branch (very small) coverage. Would be nice to catch them up so they don't fall further behind.

Reply:
Did some minor restructuring that covers those branches and hopefully improves readability somewhat.

Reply:
Just did a sanity run there are a couple more annoying `?.` in `protocol-authorization.ts`, interfaces/protocols-`query.ts` and `interfaces/records-read.ts`.

And a more real block in `handlers/permissions-revoke.ts`.

Fine if you'd rather do this in a follow up PR.

Reply:
I'll grab em in a follow up PR